docs/source/intervention_models/insecticide_treated_nets/index.rst

```diff
@@ -62,58 +62,26 @@ Insecticide treated bed nets effectively reduce malaria incidence (and therefore
 
-.. list-table:: Ethiopia baseline coverage data: proportion of pregnant women sleeping under insecticide treated nets
-  :widths: 10 15 5 5 25 
+.. list-table:: ITN Baseline Coverage
   :header-rows: 1
 
-  * - Study
-    - Population
-    - Study year
+  * - Location
+    - Subpopulation
+    - Coverage parameter
     - Value
     - Note
-  * - [Belay-and-Deressa-2008]
-    - 814 pregnant women from northern Ethiopia
-    - 2006
-    - 0.345
-    - District purposely selected to represent amalrious areas of southern Tigray in northern Ethiopia. Seven rural and one urban sampling unit.
-  * - [Deressa-et-al-2014]
-    - 55 pregnant women from 1235 households from 16 randomly selected villages in southern Ethiopia
-    - 2008
-    - 0.60
-    - ITN use among pregnant women positively associated with household wealth index
-  * - [Yitayew-et-al-2018]
-    - 226 pregnant women who attended antenatal clinics at the Adis Zemen hospital
-    - 2018
-    - 0.708
-    - ITN use positively associated with maternal education and maternal age
-  * - [Fuge-et-al-2015]
-    - 398 pregnant women in Hadiya Zone, Shashogo District, Ethiopia
-    - 2014
-    - 0.082
-    - 
-  * - [Graves-et-al-2011]
-    - X
-    - 2006-2007
-    - X
-    - 
-  * - [Negash-et-al-2012]
-    - Six districts in Afar National Regional State in northeastern Ethiopia
-    - 2008
-    - 0.70
-    - Observed use (reported use was 79.1%)
-  * - [Ouedraogo-et-al-2019]
-    - 2784 women who had pregnancy outcome in the year preceding the survey
-    - 2016
-    - 0.264
-    - reported "always" sleeping under the net during their pregnancy. ITN use was positively associated with attending at least one ANC visit.
+  * - Ethiopia
+    - Pregnant women in malarious areas
+    - Proportion sleeping under ITNs
+    - 0.44 (SD: 0.021)
+    - Assume normal distribution of uncertainty. SD of uncertainty distribution calculated from :math:`SE = \sqrt{p * (1 - p) / n}`. Data from Ethiopia MIS 2015.
 
-.. todo::
-
-  Determine baseline coverage value to be used for Ethiopia, either by meta-analysis or by selection of large/representative recent study
+.. note::
 
-  NOTE: GBD estimates ITN coverage. Use this value instad after discussion with modelers.
+  Because we do not currently model malarious areas in our simulations (but rather generalize the malaria burden to the entire population), we will assume that the coverage proportion of ITNs in malarious areas is representative of the general population of pregnant women as well.
```

Comment:
This will likely lead to a dramatic over-estimate of the impact of ITN scale-up in Ethiopia, so let's get an updated number for the national coverage at some point.  By eye, I would say something 10x smaller, see https://malariaatlas.org/research-project/metrics-of-insecticide-treated-nets-distribution/exceedance-surfaces/

Reply:
Part of the difference between the figure here and the one you linked is because use in the general population is lower than among pregnant women. The MIS survey I cited has use in the general population in malarious areas at 38% vs. 70% of pregnant women. Maybe we could derive a crosswalk value for pregnant women from MIS and then apply it to an updated national value?

Reply:
Thinking through it again, though, I wonder if we could derive some fraction for the proportion of the population living in "malarious areas" and then only apply the intervention to that fraction? The effect size of the intervention comes from areas with high malaria burden, so we will need to make some decisions/assumptions about the burden of malaria in Ethiopia. I will follow up about this!

Reply:
Good strategy.  It is possible that this fraction is already available as an ancillary input for the MAP team's work.

Reply:
I made an update using just the MIS data and associated assumptions in this PR, but I will keep looking into the MAP work and finding the best data source for this on my todo list to revisit after finalizing the wasting treatment model first. https://github.com/ihmeuw/vivarium_research/pull/625